types/grecaptcha/index.d.ts

```diff
@@ -8,6 +8,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare var grecaptcha: ReCaptchaV2.ReCaptcha;
+export = grecaptcha;
```

Comment:
This type definition is intended to map the client script of Google reCaptcha.
The script's API is provided as a global variable, not as a module.

Reply:
I suppose that makes sense. I had quite a few issues trying to access it properly while still having the standard grecaptcha (non-ts library) installed. I think the var names were duplicate in the global namespace from both the types and the non-typed.

Side note: I could be totally wrong, I just had issues with this :)